tabpy/models/scripts/SentimentAnalysis.py

```diff
@@ -3,8 +3,12 @@
 from nltk.sentiment.vader import SentimentIntensityAnalyzer
 import sys
 from pathlib import Path
-sys.path.append(str(Path(__file__).resolve().parent.parent.parent / 'models'))
-from utils import setup_utils
+from tabpy.models.utils import setup_utils
+
+
+import ssl
```

Comment:
Why is this necessary only for Sentiment Analysis?

Reply:
nltk package requires dictionaries which are not installed with the package. And the download is done with HTTPS where the server has untrusted cert. To prevent connection failure the code tells to not validate a cert.